script.js

```diff
@@ -0,0 +1,16 @@
+import http from "k6/http";
```

Comment:
GET route looks good. I think the POST route can also be written in this file if you are planning to do that next.